syz-ci/syz-ci.go

```diff
@@ -144,6 +144,12 @@ type Config struct {
 	// Per-vm type JSON diffs that will be applied to every instace of the
 	// corresponding VM type.
 	PatchVMConfigs map[string]json.RawMessage `json:"patch_vm_configs"`
+	// Some commits don't live long. Cache them somewhere to increase lifetime.
+	// The cache is used by coverage merger.
+	GitCache string `json:"git_cache"`
+	// "git config" params you want to be called to configure git cache.
+	// Only one git config command will be called.
+	GitCacheConfig []string `json:"git_cache_config"`
```

Comment:
Is it meant to contain e.g. the credentials? If yes, could we configure it on the machine itself rather than do it in Go via the syz-ci config?

Reply:
Do we have a machine configuration scripts for every ci?

Reply:
It is open data -> https://cloud.google.com/secure-source-manager/docs/use-git#set_up_your_credentials.

```
git config credential.'https://*.*.sourcemanager.dev'.helper gcloud.sh
```

Reply:
We can make it a part of syzbot docker file. I guess...

Reply:
> We can make it a part of the syzbot docker file. 

Sounds good (if that is possible).

Reply:
PTAL

Reply:
Pushed container.